CONTRIBUTING.md

```diff
@@ -0,0 +1,119 @@
+## Development of web-ext
+
+To get started, first install it from source as documented in [readme](https://github.com/mozilla/web-ext/blob/master/README.md).
```

Comment:
I like explicitly linking to it in this way. It makes it easier if they begin by filing an issue which will show them a link to this doc and that might be the first time they have read it.